cli/core/status.go

```diff
@@ -73,10 +73,14 @@ func GetStatus(ctx context.Context, eigenpodAddress string, eth *ethclient.Clien
 	state, err := beaconClient.GetBeaconState(ctx, "head")
 	PanicOnError("failed to fetch state", err)
 
-	allValidators := FindAllValidatorsForEigenpod(eigenpodAddress, state)
-	activeValidators := SelectActiveValidators(eth, eigenpodAddress, allValidators)
+	allValidators, err := FindAllValidatorsForEigenpod(eigenpodAddress, state)
+	PanicOnError("failed to find validators", err)
```

Comment:
should this (and the ones below it) return an error rather than panic?

Reply:
`status` isn't really part of the core libraries. doesn't generate any proofs

Reply:
figured i'd just leave it for now i'll clean it up if we need it